example/src/main/java/io/realm/examples/adapters/model/Counter.java

```diff
@@ -15,19 +15,30 @@
  */
 package io.realm.examples.adapters.model;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
 import io.realm.RealmObject;
 
-public class TimeStamp extends RealmObject {
+public class Counter extends RealmObject {
+    public static final String FIELD_COUNT = "count";
+
+    private static AtomicInteger integerCounter = new AtomicInteger(0);
```

Comment:
I believe this is in order to have a global counter while the app is running so the list will have ever increasing numbers.